File: examples/javascript/constructs/for_loop_control.js
// Ported from examples/lumen/constructs/for_loop_control.lm by scripts/port_examples.py; edit the Lumen original, not this file.
let i = 0;
while (i < 15) {
    if (i === 10) {
        break;
    }
    process.stdout.write(String(i));
    if (i < 9) {
        process.stdout.write(", ");
    }
    i = i + 1;
}
console.log("");

File: examples/python/constructs/for_loop_control.py
import sys
# Ported from examples/lumen/constructs/for_loop_control.lm by scripts/port_examples.py; edit the Lumen original, not this file.
for i in range(0, 15):
    if i == 10:
        break
    sys.stdout.write(str(i))
    if i < 9:
        sys.stdout.write(", ")
print("")
